app/services/doc_auth/error_generator.rb

```diff
@@ -109,12 +109,17 @@ def get_id_type_errors(classification_info)
       %w[Front Back].each do |side|
         side_class = classification_info.with_indifferent_access.dig(side, 'ClassName')
         side_country = classification_info.with_indifferent_access.dig(side, 'CountryCode')
+        side_issuer_type = classification_info.with_indifferent_access.dig(side, 'IssuerType')
+
         side_ok = !side_class.present? ||
                   SUPPORTED_ID_CLASSNAME.include?(side_class) ||
                   side_class == 'Unknown'
         country_ok = !side_country.present? || supported_country_codes.include?(side_country)
-        both_side_ok &&= side_ok && country_ok
-        error_result.add_side(side.downcase.to_sym) unless side_ok && country_ok
+        issuer_type_ok = !side_issuer_type.present? ||
```

Comment:
Is this first line saying, "the issuer type is okay if there is no `side_issuer_type` present?

Reply:
Yes.

Reply:
I think I'm not following the reasoning behind this. I would have expected needing a `side_issuer_type` for it to be "ok". Do you mind explaining further why not having a `side_issuer_type` means it is okay?

Reply:
@dawei-nava a couple of reasons. When a document cannot be classified with a known document type or issuer type etc. It may be because other issues like image quality issues, this is to allow these errors to pop up. Also, the authentication will fail for sure and will be caught, the chance of supported document type(etc) with bad image quality is classified as unknown exists. So for individual checks, only rejects on definitive negative gives us more flexibility without affecting the vendor doc auth results.

Reply:
I see - thank you for clarifying!